rubicon_ml/domain/mixin.py

```diff
@@ -25,3 +25,18 @@ def remove_tags(self, tags: List[str]):
             A list of string tags to remove from this domain model.
         """
         self.tags = list(set(self.tags).difference(set(tags)))
+
+
+class CommentMixin:
+    """Adds comment support to a domain model."""
+
+    def add_comments(self, comments: List[str]):
+        """
+        Add new comments to this model.
+
+        Parameters
+        ----------
+        comments : List[str]
+            A list of string comments to add to the domain model.
+        """
+        self.comments = list(set(self.comments).union(set(comments)))
```

Comment:
this was originally in tags because we didn't want duplicate tags. comments are pretty unstructured, so I don't think its necessary to guard against duplicates here. let's just do `self.comments.extend(comments)`

Reply:
I'll remove it, but just for the sake of curiosity - is there ever a situation we would actually want duplicates? Even if the check isn't as necessary, does having it actually hurt anything?

Reply:
I could imagine multiple users having a conversation and there being multiple "yes"s, "no"s. etc. but then again we'd probably want to make these more robust (like including authors) if we really wanted to support conversations like that

also, the list/union/list conversion is O(n) so it could become slow. but that'd require a lot of comments